<?xml version="1.0"?>
<launch>

	<arg name="marker_size" default="2.9" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />
	<arg name="cam_image_topic" default="/head_kinect/rgb/image_color" />
	<arg name="cam_info_topic" default="/head_kinect/rgb/camera_info" />	
	<arg name="output_frame" default="/head_kinect_rgb_optical_frame" />
	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />

    <node
        name="recognized_objects_publisher"
        pkg="velma_scripts" type="recognized_objects_publisher.py" output="screen">
    </node>
</launch>
